runtime/linux/proc/process.go

```diff
@@ -19,66 +19,12 @@
 package proc
 
 import (
-	"context"
-	"io"
-	"sync"
-	"time"
-
-	"github.com/containerd/console"
 	"github.com/pkg/errors"
 )
 
 // RuncRoot is the path to the root runc state directory
 const RuncRoot = "/run/containerd/runc"
 
-// Stdio of a process
-type Stdio struct {
-	Stdin    string
-	Stdout   string
-	Stderr   string
-	Terminal bool
-}
-
-// IsNull returns true if the stdio is not defined
-func (s Stdio) IsNull() bool {
-	return s.Stdin == "" && s.Stdout == "" && s.Stderr == ""
-}
-
-// Process on a linux system
```

Comment:
Based on the intent (to be platform-agnostic interface) this comment should be changed from "Process on a linux system".